src/test/java/io/github/oliviercailloux/y2018/apartments/valuefunction/ApartmentValueFunctionTest.java

```diff
@@ -79,15 +79,19 @@ void checkValue() {
     assertEquals(3d, lvf.getInterval().upperEndpoint());
   }
 
-  /** Function to test the computing of the subjective value of an apartment */
+  /**
+   * Function to test the computing of the subjective value of an apartment To compute the
+   * subjective value of an apartment, here is the formula : sum(attributeSubjectiveValue *
+   * attributeWeight)/sum(weight)
+   */
   @Test
   void apartmentValueFunctionTest() {
 
     assertEquals(0.5, valueFunction.getSubjectiveValue(a), 0.0001);
 
-    valueFunction.setTeleSubjectiveValueWeight(10d);
-    assertEquals(10d, valueFunction.getSubjectiveValueWeight(Criterion.TELE));
-    assertEquals(0.04587, valueFunction.getSubjectiveValue(a), 0.00001);
+    valueFunction.setTeleSubjectiveValueWeight(4.1d);
+    assertEquals(4.1d, valueFunction.getSubjectiveValueWeight(Criterion.TELE));
```

Comment:
Ceci est trop simple pour échouer et réduit la visibilité de la vraie préoccupation de ce test. OPT.

Reply:
Fix in commit 9e4419b149f12124aed5edb5611f61ca9ff4b3de